docs/language/operators/README.md

```diff
@@ -685,8 +685,8 @@ time                        uid
 
 |                           |                                                 |
 | ------------------------- | ----------------------------------------------- |
-| **Description**           | Add/update fields based on the results of an expression         |
-| **Syntax**                | `put <field> := <expression> [, <field> := <expression> ...]`   |
+| **Description**           | Add/update fields based on the results of an expression<br><br>The `put` keyword is optional, though its explicit use may be appropriate to disambiguate from where `:=` is used for [naming fields to hold aggregation results](../aggregate-functions/README.md#field-naming). |
+| **Syntax**                | `[put] <field> := <expression> [, <field> := <expression> ...]`   |
```

Comment:
You can also just have a call here where a field with the name of the call is create: `to_lower(x)`

Reply:
So a put is a list of either `<field> := <expression>` or `<call_expression>`